server/functional-test/src/test/java/org/zanata/feature/document/MultiFileUploadTest.java

```diff
@@ -49,15 +54,15 @@
     public CleanDocumentStorageRule documentStorageRule =
             new CleanDocumentStorageRule();
     private TestFileGenerator testFileGenerator = new TestFileGenerator();
-    private String documentStorageDirectory;
+    private static String testString = "Test text 1";
 
     @Before
     public void before() {
         new BasicWorkFlow().goToHome().deleteCookiesAndRefresh();
         new LoginWorkFlow().signIn("admin", "admin");
         new ZanataRestCaller().createProjectAndVersion("multi-upload",
                 "multi-upload", "file");
-        documentStorageDirectory = CleanDocumentStorageRule
+        String documentStorageDirectory = CleanDocumentStorageRule
                 .getDocumentStoragePath().concat(File.separator)
                 .concat("documents").concat(File.separator);
         if (new File(documentStorageDirectory).exists()) {
```

Comment:
It's just a warning in case CleanDocumentStorageRule fails to do its job - which may be better in the actual rule but I'll look into that later (the check is in many tests)